css/style.css

```diff
@@ -0,0 +1,124 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Inter', sans-serif;
+}
+
+a {
+  color: black;
+  text-decoration: none;
+}
+
+.wrapper {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.hero {
+  background-image: url(../img/header-illsutration-mobile@2x.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  background-position-y: 5px;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+
+.navigation {
+  margin: 0;
+}
+
+.navigation ul {
+  list-style: none;
+  margin: 0;
+  padding: 15px 45px 15px 45px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.menu-btn {
+  margin-top: 5px;
+}
+
+.menu-icon .logo {
+  font-style: normal;
+  font-weight: 600;
+  line-height: 28px;
+  color: black;
+  font-size: 22px;
+}
+
+.bold-text {
+  margin-top: 40px;
+  font-weight: 700;
+  text-align: center;
+  font-size: 30px;
+  line-height: 52px;
+}
+
+.b-text-1 {
+  color: #172b4d;
+}
+
+.b-text-2 {
+  color: #36b37e;
+  margin-top: -30px;
+}
+
+.small-text {
+  color: #172b4d;
+  font-weight: 400;
+  text-align: center;
+  font-style: normal;
+  line-height: 28px;
+  padding: 0 40px 0 40px;
+}
+
+.social-icons {
+  display: flex;
+  column-gap: 21px;
+  justify-content: center;
+  margin: 0;
+  padding-left: 0;
+}
+
+.social-icons li {
+  list-style: none;
+}
+
+.social-icons .fa {
+  color: #505f79;
+  font-size: 22px;
+}
+
+.scroll {
+  margin: 50px 0 0 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* .scroll-circle {
+    height: 40px;
+    width: 40px;
+    border: 3px solid #DFE1E6;
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+} */
+
```

Comment:
- [ ] Please make sure you remove commented code from your project to make your code clean. :pray:

Reply:
The irrelevant codes (comments) has been removed. Thank you